pages/mobile/home.py

```diff
@@ -22,6 +27,29 @@ def __init__(self, base_url, selenium):
     def header_text(self):
         return self.selenium.find_element(*self._header_locator).text
 
+    def is_user_logged_in(self):
+        self.open_menu()
+        return self.selenium.find_element(*self._sign_out_menulocator)
+
+    def is_user_logged_out(self):
+        self.open_menu()
+        return self.selenium.find_element(*self._sign_in_menu_locator)
+
+    def sign_in(self, username, password):
+        self.open_menu()
```

Comment:
I think `open_menu` needs to wait for the menu to finish animating. It appears that the menu width is 270 once it's fully open, perhaps we can wait for this?